jre_emul/android/platform/libcore/luni/src/test/java/libcore/java/util/zip/ZipFileTest.java

```diff
@@ -16,8 +16,9 @@
 
 package libcore.java.util.zip;
 
-/* J2ObjC removed.
+/* J2ObjC modified.
 import android.system.OsConstants; */
+import static libcore.io.OsConstants.*;
```

Comment:
Nit: since we only need `Libcore.os.lseek` and `Libcore.os.SEEK_CUR`, it may be better we just static-import those two symbols, instead of importing all with `*`.

Reply:
changed the static import for SEEK_CUR. I think lseek() requires `import libcore.io.Libcore;`